ambuda/views/proofing/page.py

```diff
@@ -129,8 +129,8 @@ def edit_post(project_slug, page_slug):
             conflict = cur.revisions[-1]
             form.version.data = cur.version
 
-    has_edits = bool(cur.revisions)
-    is_r0 = cur.status.name == SitePageStatus.R0
+    _ = bool(cur.revisions)
```

Comment:
here, let's delete the first line and pass `is_r0` into the template as is done in the function above.